test/project_types/script/layers/infrastructure/languages/other_project_creator_test.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Layers::Infrastructure::Languages::OtherProjectCreator do
+  include TestHelpers::FakeFS
+
+  let(:context) { TestHelpers::FakeContext.new }
+  let(:fake_capture2e_response) { [nil, OpenStruct.new(success?: true)] }
+
+  let(:type) { "payment-methods" }
+  let(:language) { "other" }
+  let(:domain) { "fake-domain" }
+  let(:project_name) { "myscript" }
+  let(:sparse_checkout_repo) { nil }
+  let(:sparse_checkout_branch) { "fake-branch" }
+  let(:sparse_checkout_set_path) { "#{domain}/#{language}/#{type}/default" }
+
+  let(:config_file_content) do
+    <<~END
+      ---
+      version: '2'
+      title: #{type} script
+      description: #{type} script in other language
+      configuration:
+      type: object
+      fields: {}
+    END
+  end
+
+  let(:metadata_file_content) do
+    "{\"schemaVersions\":{\"#{type}\":{\"major\":1,\"minor\":0}},\"flags\":{\"use_msgpack\":true}}"
+  end
```

Comment:
These are just copies of what the OtherCreator outputs right now.
I'm not sure this is worth testing at the moment but I wanted to setup the skeleton for it.